recipe/meta.yaml

```diff
@@ -121,6 +123,11 @@ requirements:
     - ninja
     - python
   host:
+{% if cuda_major == 12 %}
+    - cuda-cudart-dev
```

Comment:
`cuda-cudart-dev` should only be needed if we are dynamically linking to it. Are we dynamically linking to `libcudart` or is it statically linked?

Asking in part as [`nvcc` defaults to static linking `cudart`]( https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#cudart-none-shared-static-cudart )

Reply:
CMakeLists are using the `CUDA::cudart` target not the `CUDA::cudart_static` target, so I assume dynamic linking.

https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html?highlight=cuda+cudart#cuda-runtime-library